src/components/SidePanes/UsersPane.jsx

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { Icon, InputSelect, Alert } from '@edx/paragon';
+import Select, { components } from 'react-select';
+import ReactTooltip from 'react-tooltip';
+
+import { Icon, Alert } from '@edx/paragon';
+import { Add } from '@edx/paragon/icons';
 
 import Pane from './Pane';
 import User from './User';
-import FieldLabel from '../FieldLabel';
+
+// customize react-select dropdown options to show tooltips
+const Option = (props) => (
+  <div className={`tooltip-${props.data.value}`} data-tip data-for={`tooltip-${props.data.value}`}>
+    <components.Option {...props} className={`option-${props.data.value}`} />
+    <ReactTooltip id={`tooltip-${props.data.value}`}>
+      <span>{props.data.label}</span>
+    </ReactTooltip>
+  </div>
+);
```

Comment:
Suggestion: work with paragon group to add this in paragon lib itself.

Reply:
+1 to this @zawan-ila